models/currency.js

```diff
@@ -0,0 +1,189 @@
+const firestore = require('../utils/firestore')
+const walletModel = firestore.collection('wallets')
+const bankModel = firestore.collection('banks')
+const currencyExchangeModel = firestore.collection('currencyExchange')
+// const transactionModel = firestore.collection('transactions')
+
+const allCurrencyName = 'allCurrencies'
+
+/**
+ * Fetch the currency exchange rates
+ * @return {Promise<exchangeData>}
+ */
+const fetchExchangeRates = async () => {
+  try {
+    const exchangeSnapshot = await currencyExchangeModel.doc(allCurrencyName).get()
```

Comment:
Don't we pass the document ID in `.doc()`? Are we going to manually set the document ID as `allCurrencies`?